### PR TITLE
Add a ClosestEdgeCell and a ClosestMatchingEdgeCell function to lua

### PR DESCRIPTION
**Pull request by abcdefg30**
Makes use of the new methods introduced in #10851.

### CODE REVIEW

**Comment by obrakmann** [comment]
:+1:

File: mods/ra/maps/allies-01/allies01.lua [hunk 1]
```diff
@@ -19,6 +19,16 @@ end
 SendJeeps = function()
 	Reinforcements.Reinforce(player, JeepReinforcements, InsertionPath, DateTime.Seconds(2))
 	Media.PlaySpeechNotification(player, "ReinforcementsArrived")
+
+	-- TESTCASE
```
Comment (pchote) [inline mods/ra/maps/allies-01/allies01.lua]: This testcase doesn't seem to work.  Where are the dogs supposed to spawn?

Comment (pchote) [inline mods/ra/maps/allies-01/allies01.lua]: Ah, nevermind.  I didn't realize that you got jeep reinforcements later in the mission.

**Comment by pchote** [comment]
:+1: after you remove the testcase.

**Comment by Phrohdoh** [comment]
Another [fixup](https://github.com/Phrohdoh/OpenRA/tree/lua-edge-cells) if we get to this before @abcdefg30 returns.

**Comment by abcdefg30** [comment]
Removed the testcase.

**Comment by Phrohdoh** [comment]
:+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/72505d7fcbf7c71ab1ff19382c5a484929c0bd7a)

**Pull request merged**